Polishchuk_Dariia_HW2(3 reverse).cpp

```diff
@@ -0,0 +1,15 @@
+#include <iostream>
+
+int main() {
+  int number{};
+  int reverse_number = 0;
```

Comment:
```suggestion
  int reverse_number{};
```
это будет тоже самое что и вы написали